domains/vrbz.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "vrbz",
+           "email": "108748279+vrbz@users.noreply.github.com",
+           "discord": "590921984020447235"
+        },
+    
+        "record": {
+            "CNAME": "3e898fe9847779bdcaea77cb3b4fcef5.is-a.dev"
```

Comment:
3e898fe9847779bdcaea77cb3b4fcef5.is-a.dev ofc doesnt exist, please change it to an actual valid CNAME record

Reply:
> if i attempt to add the subdomain it tells me i first need to link the actual domain to be able to add subdomains, so im not sure if its possible to use with that service

if its asking you to connect is-a.dev and then vrbz.is-a.dev then yea it isnt possible, if you want to you can try github pages or cloudflare pages